### ISSUE TITLE
Specify location for serverCache.json, create option for serverCache update frequency, allow for local server list

### ISSUE THREAD

**Issue by begleysm** [begleysm/ipwatch#13]
Currently serverCache.json is saved to wherever you run the program from.  This could be the location of ipwatch.py, it could be the home directory, or somewhere else.  This should be normalized and probably made an option in the config.

Currently the update frequency of serverCache.json is hard coded to 90 days.  This should probably be made as a config option and should likely default to a more frequent update rate.

There should be an option to use a local server list instead of forcing the SW to use the list from github (and/or overwrite a custom user list when it is updated).

**Comment by wushuzh** [comment]
:+1:

When I enter the directory of ipwatch and execute it continuously, it works well, using serverCache.json without need of github.

But when it triggers by crontab, due to it cannot find the location of serverCache.json, it tries to download from github every time.
```
00 * * * * python3 /opt/ipwatch/ipwatch.py /opt/ipwatch/config.txt
```

**Comment by begleysm** [comment]
Yeah I was just noticing the other day that tasks run by a crontab execute "from" the root user home folder at `/root`.  This is where I found my serverCache.json file.  As I mentioned above, I think I'll add a serverCache.json save location to the config file soon.

However, I'm surprised to hear you say it "tries to download from github every time."  I would have expected it to be in your `/root` folder (and then not have to download after that) as well.
